backend-tests/tests/test_devauth_v2.py

```diff
@@ -1224,3 +1224,135 @@ def compare_aset(authset, api_authset):
        assert authset.id_data == api_authset['identity_data']
        assert util.crypto.rsa_compare_keys(authset.pubkey, api_authset['pubkey'])
        assert authset.status == api_authset['status']
+
+class TestDefaultTenantTokenEnterprise(object):
+
+    uc = ApiClient(useradm.URL_MGMT)
+    devauthd = ApiClient(deviceauth_v1.URL_DEVICES)
+    devauthm = ApiClient(deviceauth_v2.URL_MGMT)
+
+    def test_default_tenant_token(self, clean_mongo):
+        """Connect with a device without a tenant-token, and make sure it shows up in the default tenant account"""
+
+        default_tenant = create_tenant('default-tenant')
+
+        # Restart the deviceauth service with the new token belonging to `default-tenant`
+        deviceauth_cli = CliDeviceauth()
+        deviceauth_cli.add_default_tenant_token(default_tenant.tenant_token)
+
+        # Create a user for the default tenant
+        default_tenant_user = create_user('root@admin.com', 'foobarbaz', default_tenant.id)
+        r = self.uc.call('POST',
+                    useradm.URL_LOGIN,
+                    auth=(default_tenant_user.name, default_tenant_user.pwd))
+        assert r.status_code == 200
+        default_utoken = r.text
+
+        # Create a device, and try to authorize
+        create_random_authset(default_utoken, default_tenant.tenant_token)
```

Comment:
if I understand correctly you are trying to test if the device with empty tenant token will appear in the default tenant account, but you've created device with tenant token (not empty, but regular, valid one)

Reply:
Yes! good catch.